operator/config/rbac/auth_proxy_client_clusterrole.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: metrics-reader
+rules:
+- nonResourceURLs:
+  - "/metrics"
```

Comment:
which metrics are we expected to scrape?

Reply:
Does the agent expose it to external endpoints, i.e., prometheus or does it consume any metric? If it just exposes them it does not requires get access to any `/metrics`

Reply:
I don't think the agent consumes any of those.

We only expose them.

Reply:
Please see this comment
https://github.com/gravitational/teleport/pull/13331#issuecomment-1161460367